src/saml2/attributemaps/saml_uri.py

```diff
@@ -375,11 +413,14 @@
         'teletexTerminalIdentifier': X500ATTR_OID+'22',
         'telexNumber': X500ATTR_OID+'21',
         'title': X500ATTR_OID+'12',
+        'transactionIdentifier': DIGG_OID+'2',
         'uid': UCL_DIR_PILOT+'1',
         'uniqueMember': X500ATTR_OID+'50',
         'userCertificate': X500ATTR_OID+'36',
+        #  'userCertificate': DIGG_OID+'10',
```

Comment:
so, this is a bummer and it's not the only problematic attribute `FriendlyName` (see #489).

Unfortunately, there is no way to support all of them at once. If you need that you need to provide your own maps and have the proper configuration in place (https://pysaml2.readthedocs.io/en/latest/howto/config.html#attribute-map-dir)

For now I think it's good to merge this. Going forward we should [move away from `FriendlyName`s](https://github.com/IdentityPython/pysaml2/issues/549)